Eigenvalues/InversePowerMethod.m

```diff
@@ -0,0 +1,34 @@
+% [USES] Ad-hoc/SIT
+% [USES] Ad-hoc/SST
```

Comment:
Are deja Doolittle incluse SIT si SST, nu mai trebuie sa le referentiezi tu ca tu nu le folosesti explicit in cod.